frontend-opal/.eslintrc.json

```diff
@@ -25,8 +25,9 @@
     "parser": "babel-eslint",
     "rules": {
 	"react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-
 	"no-use-before-define": "warn",
-	"no-undef": "warn"
+    "no-undef": "warn",
+    "import/no-unresolved": "warn",
+    "no-unused-vars": "warn"
```

Comment:
@hondahmd maybe eslint should check the json files too :D
https://github.com/green-fox-academy/JSA-Opal-ShenZhen/blob/bdb9b7599b906d2166387f23e0b24aca838d3a28/frontend-opal/.eslintrc.json